about.html

```diff
@@ -0,0 +1,198 @@
+            <a href="home.html" class="logo-txt"
+              >F<i class="fa-brands fa-google"></i> MusicFest</a
+            >
+          </div>
+          <div class="bx2">
+            <a href="home.html">Home</a>
+            <a href="about.html">About</a>
+            <a href="#">Venue</a>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <section class="first-sec">
+      <div class="first-sec-container">
+        <div class="first-sec-heading">
+          <h3>"Be a Part Musical History"</h3>
+          <h1>Festus Group Music Festival</h1>
+        </div>
+        <div class="description">
+          <p>
+            FG Music Festival is an alternative music festival that brings the
+            newest, most interesting and mind-blowing artists to Ghana. FG Music
+            Festival will liberate you from your everyday surroundings we will
+            welcome artists to Ghana, whom you might never even heard of, but
+            have always wanted to discover. The artists who you will see are
+            true to the music in their hearts, mainstream is not something they
+            are drawn to. We will take a ride on the wild side and we are
+            inviting all of you to join us to enjoy the freedom the
+            independence. Get in touch with us!
+          </p>
+        </div>
+        <div class="date">
+          <p>
+            Please contact us via Email for futher questions about FG Music
+            Festival 2022. <br />
+          </p>
+          <h4>enningfestus@gmail.com</h4>
+        </div>
+      </div>
+    </section>
+    <section class="second-sec">
+      <div class="second-sec-container">
+        <div class="second-sec-heading">
+          <h2>FG Music Festival 2022 Logo</h2>
+          <div class="underline"><hr /></div>
+        </div>
+        <div class="date1">
+          <p>
+            The Logo of FG Music Festival is ddesigned by me by combining texts
+            and font-awesome icons.
+          </p>
+        </div>
+        <div class="description1">
+          <div class="desc-wrapper">
+            <div class="bx1">
+              <a href="home.html" class="logo-txt"
+                >F<i class="fa-brands fa-google"></i> MusicFest</a
+              >
+              <div class="date-wrapper"><h6>2022.11.26(Sat)</h6></div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <hr />
+
+    <section class="third-sec-container">
+      <div class="third-sec-headline">
+        <div class="title-wrapper">
+          <h2>See the past CC Global Summits</h2>
+          <div class="underline"><hr /></div>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Accusantium
+            facilis neque sed aliquam reprehenderit consequatur.
+          </p>
+        </div>
+      </div>
+
+      <div class="card-container">
+        <div class="card">
+          <img class="img1" src="./img/bg1.jpg" alt="" />
```

Comment:
- [ ] Kindly ensure all your images contain alt attributes, this is important so when browsers are unable to render a page properly a text is displayed 👍🏿.